sdk/ml/azure-ai-ml/azure/ai/ml/operations/_data_operations.py

```diff
@@ -314,6 +320,67 @@ def create_or_update(self, data: Data) -> Data:
                     )
             raise ex
 
+    @experimental
+    def import_data(self, data_import: DataImport) -> Job:
+        """Returns the data import job that is creating the data asset.
+
+        :param data_import: DataImport object.
+        :type data_import: azure.ai.ml.entities.DataImport
+        :return: data import job object.
+        :rtype: ~azure.ai.ml.entities.Job
+        """
+
+        experiment_name = "data_import_" + data_import.name
+        component_name = (
+            "import_data_database" if isinstance(data_import.source, Database) else "import_data_file_system"
+        )
+        data_import.type = AssetTypes.MLTABLE if isinstance(data_import.source, Database) else AssetTypes.URI_FOLDER
+        if "{name}" not in data_import.path:
+            data_import.path = data_import.path.rstrip("/") + "/{name}"
+        import_job = import_data(
+            description=data_import.description or experiment_name,
+            tags=data_import.tags,
+            display_name=experiment_name,
+            experiment_name=experiment_name,
+            compute="serverless",
+            source=data_import.source,
+            outputs={
+                "sink": Output(
+                    type=data_import.type, path=data_import.path, name=data_import.name, version=data_import.version
+                )
+            },
+            is_deterministic=False,
+            component="azureml://registries/azureml-dev/components/" + component_name + "/versions/1",
```

Comment:
we can leave this empty here,  there is an default value in implementation of import_data